contrib/systemd/traefik.service

```diff
@@ -1,11 +1,34 @@
 [Unit]
 Description=Traefik
+Documentation=https://docs.traefik.io
+After=network-online.target
+AssertFileIsExecutable=/usr/bin/traefik
+AssertPathExists=/etc/traefik.toml
```

Comment:
Please refer to the documentation https://docs.traefik.io/basics/#configuration-file

`/etc/traefik.toml` is not the default path.

> By default, Traefik will try to find a traefik.toml in the following places:
> - /etc/traefik/
> - $HOME/.traefik/
> - `.` the working directory

Reply:
Fixed.